tests/src/test/java/org/jboss/windup/tests/bootstrap/migrate/ExportCsvTest.java

```diff
@@ -33,5 +39,40 @@ public void exportCsv() throws IOException {
 
         File appTagsCsv = new File(tmp.getRoot(), "ApplicationFileTechnologies.csv");
         assertTrue(appTagsCsv.exists());
+
+        List<List<String>> records = readCSVFile(appTagsCsv);
+        records.forEach(record ->
+                {
+                   assertTrue(isRowSorted(record));
+                }
+        );
+    }
+
+    private List<List<String>> readCSVFile(File csvFile) throws IOException
+    {
+        List<List<String>> rows = new ArrayList<>();
+        try (BufferedReader buffer = new BufferedReader(new FileReader(csvFile))) {
+            String lineString;
+            while ((lineString = buffer.readLine()) != null) {
+                String[] columns = lineString.split(DELIMITER);
```

Comment:
I thought it was not that easy to parse a CSV .
What happens if we have : 
35, 1456, "mytext", 89000, "other,text", 45
I mean the text encloser " " should allow to have the DELIMITER inside the text ?

Reply:
Here there's one approach that could help :
https://www.mkyong.com/java/how-to-read-and-parse-csv-file-in-java/

Reply:
My only point is this is a test class, the csv we input to it is always the same one, and we know none of the fields are like this. If you think that remembering this it's still worth doing, then I will.

Reply:
Ohh, ok, if this is a controlled behavior, then it's OK.